versions/3.1.1.md

```diff
@@ -2021,7 +2021,7 @@ $response.header.Location | https://example.org/subscription/1
 
 ##### Callback Object Examples
 
-The following example uses the user provided `queryUrl` query string parameter to define the callback URL.  This is an example of how to use a callback object to describe a WebHook callback that goes with the subscription operation to enable registering for the WebHook.
+The following example uses the user provided `queryUrl` query string parameter to define the callback URL.  This is similar to a [`webhooks`](#oasWebhooks), but differs in that the callback only occurs because of the initial request that sent the `queryUrl`.
```

Comment:
```suggestion
The following example uses the user provided `queryUrl` query string parameter to define the callback URL.  This is similar to a [`webhooks`](#oasWebhooks) callback, and differs in that the callback only occurs because of the initial request that sent the `queryUrl`.
```

Reply:
I can't get behind this one.  There is an error to fix here which is that it should have been "webhook" (singular) rather than "webhooks".  I really don't want to conflate webhooks and callbacks here- that's why I reworded it in the first place.

As for `and` vs `but`, the point here is to emphasize the different (`but`) rather than the similarity (`and`).

I'll fix the plural, but I'd rather not do the others unless there's some aspect of the definitions of "webhook" and "callback" that I'm missing (_which is very possible so please do lmk if you think so)_.